src/libs/SearchUIUtils.ts

```diff
@@ -576,7 +576,7 @@ function getReportSections(data: OnyxTypes.SearchResults['data'], metadata: Onyx
 
     const reportIDToTransactions: Record<string, ReportListItemType> = {};
     for (const key in data) {
-        if (isReportEntry(key)) {
+        if (isReportEntry(key) && (data[key].type === CONST.REPORT.TYPE.IOU || data[key].type === CONST.REPORT.TYPE.EXPENSE || data[key].type === CONST.REPORT.TYPE.INVOICE)) {
```

Comment:
if you wanna be nice and tidy, you could declare at the top of file:
```
const moneyRequestReportTypes = [....]
```
then 

```suggestion
        (...) && moneyRequestReportTypes.includes(data[key].type))
```

Reply:
+1 agreement with this suggestion

Reply:
Thanks for the suggestion. I'm trying to implement it but it creates a nasty TS error that I can't seem to get rid off, no matter what I do haha
![Screenshot 2025-05-14 at 8 52 52 AM](https://github.com/user-attachments/assets/bd6d3ff0-f9f8-49ef-81b4-cbc84cb40f52)